web/libs/editor/src/components/RelationsOverlay/RelationsOverlay.jsx

```diff
@@ -206,6 +206,7 @@ class RelationsOverlay extends PureComponent {
       <AutoSizer onResize={this.onResize}>
         {() => (
           <svg className="relations-overlay" ref={this.rootNode} xmlns="http://www.w3.org/2000/svg" style={style}>
+            <title>{this.state.shouldRender ? "Arrow Marker" : ""}</title>
```

Comment:
oh, I'm not sure about it. it might mess the user interaction with tooltips, especially this can be inconvenient with lot of relations over text

Reply:
ah, the whole svg has `pointer-events: none` so it's good to go with title!